doc/checklists/interpolating_initial_conditions.md

```diff
@@ -0,0 +1,58 @@
+# Steps for interpolating out-of-the-box initial conditions
```

Comment:
Hmmm. I'm not sure I see the idea of having a directory for checklists? I suppose if they go under docs it makes sense. But, in some cases shouldn't the checklist be IN the directory where it applies? The checklist for making surface datasets would then be in the tools/mksurfdata_map directory, because that's where you have to go to do that task. It's not really natural to think "I should look in the checklists directory for a checklist regarding a task that I don't know there even might be a checklist for". The checklist for bringing changes to master belongs in doc, since that's where you make the ChangeLog.

I think what I would prefer is a naming convention that makes it obvious that something is a checklist that should be followed. For example, something like...

README.CHECKLIST.<description>

So it brings your attention to it, says it's a checklist, and tells you what it's regarding...

Reply:
Based on the location where this comes into play, I'd say this checklist naturally goes under the bld/namelist_files directory right now.

Reply:
I'm happy with your idea of putting checklists where they most logically belong, but with a naming convention. In the case of this file, I don't think I'd remember that it's in bld/namelist_files, but if we have a convention that all such checklists should have CHECKLIST in their name, then I can at least do `find . -name '*CHECKLIST*'` to find it.

So what would you recommend this file be called? Are you thinking `README.CHECKLIST.interpolating_initial_conditions.md`, for example?

Reply:
Yes, exactly that's what I'm thinking. I don't know that CHECKLIST has to be in caps, but it makes it yell louder that way, and the standard README. as a prefix also brings attention to it. We could also put a note at the top that such checklists are named that way.